src/foam/comics/DAOCreateControllerView.js

```diff
@@ -131,12 +126,24 @@ foam.CLASS({
   ],
 
   listeners: [
-    function onFinished() {
-      this.stack.back();
+    {
+      name: 'onFinished',
+      on: [
```

Comment:
It would be nice if the adapt: property on 'on' would adapt a single string into an array, or even split it on ',', so we could just do on: 'data.finished' or on: 'data.finished,this.something'.

Reply:
I add support for this case.